Data/BloggerContext.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace Blogger.Data
+{
+    public class BloggerContext : DbContext
+    {
+        public DbSet<Post> Posts {set; get;}
+
+        protected override void OnConfiguring(DbContextOptionsBuilder options)
+        {
+            options.UseSqlServer("Data Source=localhost;Initial Catalog=Blogger;Integrated Security=True");
```

Comment:
Can we read this connection string from any configuration file?

Reply:
Oh! I missed that. I am on it. 👍